app/services/activities/import_from_csv.rb

```diff
@@ -239,6 +240,15 @@ def convert_gdi(gdi)
         )
       end
 
+      def convert_gcrf_challenge_area(area)
+        area = area.to_s
+        area = "Not applicable" if area == "N/A"
+
+        raise I18n.t("importer.errors.activity.invalid_gcrf_challenge_area") unless gcrf_challenge_area_options.values.map(&:to_s).include?(area)
+
+        gcrf_challenge_area_options.invert[area]
+      end
```

Comment:
I'm a little bit iffy about this. On the other fields of this nature, we accept codes, rather than text, becuase text is a bit more error prone, because of differences in punctuation, capitalisation etc. I think it might be better to be consistent.